charts/argo-cd/crds/crd-application.yaml

```diff
@@ -381,6 +465,14 @@ spec:
                       type: array
                     kind:
                       type: string
+                    managedFieldsManagers:
```

Comment:
I think this field was added in master but not in the v2.2 branch of ArgoCD.

The v2.2.2 tag version of this file doesn't seem to have this field: https://github.com/argoproj/argo-cd/blob/v2.2.2/manifests/crds/application-crd.yaml - I guess it's destined for a future 2.3 minor version release?

Reply:
I'd assume so :+1: